package.json

```diff
@@ -14,21 +14,15 @@
     "prepublish": "yarn build",
     "prettier": "prettier --write \"**/*.{scss,css,js}\"",
     "prettier:diff": "prettier --list-different \"**/*.{scss,css,js}\"",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "semantic-release":
```

Comment:
Is this from prettier?

Reply:
Yup, any formatting issues like this were from prettier